behave_django/management/commands/behave.py

```diff
@@ -106,7 +124,18 @@ def get_behave_args(self, argv=sys.argv):
         """
         parser = BehaveArgsHelper().create_parser('manage.py', 'behave')
         args, unknown = parser.parse_known_args(argv[2:])
-        return unknown
+
+        behave_args = []
+        for option in unknown:
+            # Remove behave prefix
+            if option.startswith('--behave-'):
+                option = option.replace('--behave-', '')
```

Comment:
We probably should add a `1` (`maxreplace`) parameter here too, to be on the safe side. Sorry!